### PR TITLE
Allow users to have complex logic in templates

### LINKED ISSUE #56: Add (string) comparison as helper function
Would it be possible, to include a custom helper function for Handlebars? I want to create templates based on the input upon creation. However, simple true/false evaluation doesn't do the trick for me. There are already solutions available, e.g. https://stackoverflow.com/questions/10736907/handlebars-js-else-if but I guess, that has to be included at the plugin level. In that question, one solution is to create an ifEquals function via:
```
Handlebars.registerHelper('ifEquals', function(arg1, arg2, options) {
    return (arg1 == arg2) ? options.fn(this) : options.inverse(this);
});
```

and then call it in the template via:
```
{{#ifEquals project "project 1"}}
     Project 1 selected
   {{else ifEquals project "project 2"}}
     Project 2 selected
{{/ifEquals}}
```

Additionally, I also want to change the `template_title` based on a variable or change the `template_notebook`

### PR DESCRIPTION
**Pull request by nishantwrp**
Fixes #56, #64

### CODE REVIEW

**Comment by ImperialSquid** [comment]
Looks great! You've definitely overhauled my little contribution huh? 😂😅

Just a thought, for completeness, there should probably be a power (a**b) and modulo (a mod b) operator in the maths helper, then pretty much every kind of maths you might want to do is covered!

File: src/helpers/condition.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+            case "not":
+                return !v1;
+            case "&&":
+            case "and":
+                return (v1 && v2);
+            case "||":
+            case "or":
+                return (v1 || v2);
+            default:
+                throw new Error(`Invalid comparison operator used with condition: ${operator}`);
```
Comment (nishantwrp) [inline src/helpers/condition.ts:16]: need to fix this

**Review by nishantwrp — COMMENTED** [review]
note to self

**Comment by nishantwrp** [comment]
note to self - can probably add https://github.com/joplin/plugin-templates/pull/69 too.

**Comment by dniku** [comment]
Thanks for implementing this! Any documentation on how to use it?

**Comment by nishantwrp** [comment]
Hi @dniku, this feature will be released with the next version of the plugin (sometime around end of November).

**Comment by q-wertz** [comment]
Sorry to add noise, but are there any plans for a new release?

**Comment by nishantwrp** [comment]
Hi @q-wertz, have been super busy lately, will try to get this out this month.

**Comment by q-wertz** [comment]
No problem 😊
Thanks 
And happy new year 🚀

**Comment by q-wertz** [comment]
Sorry for annoying. Any news on a new release? :see_no_evil:

**Comment by PackElend** [comment]
I'm sorry we lost attention, GSoC keeps us somehow busy.
I mentioned it again towards those who has the permission to release the new version (I'm currently don't have)

**Comment by q-wertz** [comment]
Thanks a lot 🙏😌
Have fun at GSoC :blush:

**Comment by q-wertz** [comment]
👻😿

**Comment by ed08imuj** [comment]
Any news regarding the update? I would be very interested :)

**Pull request merged**